### ISSUE TITLE
Unable to resolve play-swagger 

### ISSUE THREAD

**Issue by anisimovsergey** [iheartradio/play-swagger#122]
Hi All,

I'm trying to run the example project. I changed 
// play swagger plugin
addSbtPlugin("com.iheart" % "sbt-play-swagger" % "0.5.10-SNAPSHOT")
to 
// play swagger plugin
addSbtPlugin("com.iheart" % "sbt-play-swagger" % "0.5.1")

and tried to run update in activator console (tried package and run with a similar result). Here what I'm getting:

```
[info] Resolving com.iheart#play-swagger_2.11;0.5.1 ...
[warn] 	module not found: com.iheart#play-swagger_2.11;0.5.1
[warn] ==== local: tried
[warn]   /Users/John/.ivy2/local/com.iheart/play-swagger_2.11/0.5.1/ivys/ivy.xml
[warn] ==== public: tried
[warn]   https://repo1.maven.org/maven2/com/iheart/play-swagger_2.11/0.5.1/play-swagger_2.11-0.5.1.pom
[warn] ==== activator-launcher-local: tried
[warn]   /usr/local/Cellar/typesafe-activator/1.3.10/repository/com.iheart/play-swagger_2.11/0.5.1/ivys/ivy.xml
[warn] ==== typesafe-releases: tried
[warn]   https://repo.typesafe.com/typesafe/releases/com/iheart/play-swagger_2.11/0.5.1/play-swagger_2.11-0.5.1.pom
[warn] ==== typesafe-ivy-releasez: tried
[warn]   https://repo.typesafe.com/typesafe/ivy-releases/com.iheart/play-swagger_2.11/0.5.1/ivys/ivy.xml
[info] Resolving jline#jline;2.12.1 ...
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	::          UNRESOLVED DEPENDENCIES         ::
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	:: com.iheart#play-swagger_2.11;0.5.1: not found
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 
[warn] 	Note: Unresolved dependencies path:
[warn] 		com.iheart:play-swagger_2.11:0.5.1 ((com.iheart.sbtPlaySwagger.SwaggerPlugin) SwaggerPlugin.scala#L27)
[warn] 		  +- example:example_2.11:1.0-SNAPSHOT
[trace] Stack trace suppressed: run last *:update for the full output.
[error] (*:update) sbt.ResolveException: unresolved dependency: com.iheart#play-swagger_2.11;0.5.1: not found
```

**Comment by anisimovsergey** [comment]
I found why, I missed this line:

resolvers += Resolver.jcenterRepo 

I believe that something should be added to "Get Started"

**Comment by zhenwenc** [comment]
Thanks very much @anisimovsergey !

**Comment by kailuowang** [comment]
This is a bug in the sbt plugin which should probably add the `resolvers += Resolver.jcenterRepo` settings

**Comment by kailuowang** [comment]
fix released as 0.5.2

**Comment by mikeumus** [comment]
Putting the path in `~/.sbt/repositories` that had a version for Scala 2.10 that worked with my sbt 0.13 worked for me:

```
[repositories]
local
iheart-sbt-plugin-releases: https://dl.bintray.com/iheartradio/sbt-plugins, [organization]/[module]/(scala_[scalaVersion]/)(sbt_[sbtVersion]/)[revision]/[type]s/[artifact](-[classifier] ).[ext]
iheart-play-swagger_2.11-releases: https://jcenter.bintray.com
```

and the `jcenter` version for `sbt publishLocal`
